CHANGELOG.md

```diff
@@ -37,10 +37,11 @@
   - Fixed 'activeTab' conditions to load data for Tab-2(Registration Process) [#1050](https://github.com/eclipse-tractusx/portal-frontend/pull/1050)
 - **App Release Process**:
   - Fixed role upload does not work using Firefox [#1003](https://github.com/eclipse-tractusx/portal-frontend/pull/1003)
+- **OSP consent form**
```

Comment:
this has been added in the wrong place. Put this at the top with title "unreleased"

Reply:
Adjusted